db/migrate/20170518162225_add_status_to_booking_requests.rb

```diff
@@ -0,0 +1,10 @@
+class AddStatusToBookingRequests < ActiveRecord::Migration[5.0]
+  def change
+    add_column :booking_requests, :status, :integer, null: false, default: 0
+
+    BookingRequest.where(active: true).update_all(status: 'active')
+    BookingRequest.where(active: false).update_all(status: 'hidden')
+
+    remove_column :booking_requests, :active, :boolean, null: false, default: true
```

Comment:
Sure, figured it was good practice to include them for rollbackability, even though we're not likely to do it (and we'd have lost all the state data).